dali/test/python/test_operator_rotate.py

```diff
@@ -35,51 +35,51 @@ def get_output_size(angle, input_size, parity_correction=True):
     sina = abs(math.sin(angle))
     (h, w) = input_size[0:2]
     eps = 1e-2
-    out_w = int(math.ceil(w*cosa + h*sina - eps))
-    out_h = int(math.ceil(h*cosa + w*sina - eps))
+    out_w = int(math.ceil(w * cosa + h * sina - eps))
+    out_h = int(math.ceil(h * cosa + w * sina - eps))
     if not parity_correction:
-      return (out_h, out_w)
+        return (out_h, out_w)
 
     if sina <= cosa:
-      if out_w % 2 != w % 2:
-        out_w += 1
-      if out_h % 2 != h % 2:
-        out_h += 1
+        if out_w % 2 != w % 2:
+            out_w += 1
+        if out_h % 2 != h % 2:
+            out_h += 1
     else:
-      if out_w % 2 != h % 2:
-        out_w += 1
-      if out_h % 2 != w % 2:
-        out_h += 1
+        if out_w % 2 != h % 2:
+            out_w += 1
+        if out_h % 2 != w % 2:
+            out_h += 1
     return (out_h, out_w)
 
 
 def get_3d_lin_rotation(angle, axis):
-  # mirrors transform.h:rotation3D
-  if not angle:
-    return np.eye((3, 3), dtype=np.float32)
-  axis_norm = np.linalg.norm(axis)
-  axis = [dim / axis_norm for dim in axis]
-  u, v, w = axis
-  cosa = math.cos(angle)
-  sina = math.sin(angle)
-  return np.array([
-    [u*u + (v*v+w*w)*cosa, u*v*(1-cosa) - w*sina, u*w*(1-cosa) + v*sina],
-    [u*v*(1-cosa) + w*sina, v*v + (u*u+w*w)*cosa, v*w*(1-cosa) - u*sina],
-    [u*w*(1-cosa) - v*sina, v*w*(1-cosa) + u*sina, w*w + (u*u+v*v)*cosa],
-  ], dtype=np.float32)
+    # mirrors transform.h:rotation3D
+    if not angle:
+        return np.eye((3, 3), dtype=np.float32)
+    axis_norm = np.linalg.norm(axis)
+    axis = [dim / axis_norm for dim in axis]
+    u, v, w = axis
+    cosa = math.cos(angle)
+    sina = math.sin(angle)
+    return np.array([
+      [u * u + (v * v + w * w) * cosa, u * v * (1 - cosa) - w * sina, u * w * (1 - cosa) + v * sina],
+      [u * v * (1 - cosa) + w * sina, v * v + (u * u + w * w) * cosa, v * w * (1 - cosa) - u * sina],
+      [u * w * (1 - cosa) - v * sina, v * w * (1 - cosa) + u * sina, w * w + (u * u + v * v) * cosa],
```

Comment:
Much less readable and lines became long. With line breaks, this would become utter garbage.